content/articles/using-prime-react-in-react-applications/index.md

```diff
@@ -0,0 +1,346 @@
+                              end={<Button label="PrimeReact" icon="pi "/>}
+                          />
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+           
+        </div>
+    )
+}
+export default Navigation;
+```
+
+The list variable will store what will be displayed on our Navigation bar. 
+
+> To know more about the `Menubar` component from [here](https://primefaces.org/primereact/showcase/#/menubar)
+
+The navigation classname is used to change the pre-defined CSS styles of our React project.
+
+The above code makes our Navigation bar responsive on all device screen size.
+
+We can change the style or theme of our PrimeReact components if it's not styled. You can read more about Prime React theme from [here](https://primefaces.org/primereact/showcase/#/theming)
+
+#### Step 6 - Adding Prime React components to our project
+
+We can also import and use a ProgressBar component in our project. First we import it from primereact 
+
+```javascript
+import { ProgressBar } from 'primereact/progressbar';"
+```
+
+The above code must be placed within the app.js file, then we add the ProgressBar element inside our div tag.
+
+```javascript
+<div classname="App"> 
+ <Navigation /> 
+ <br /> 
+ <ProgressBar value={value} /> 
+</div>. 
+```
+
+This will display our progressbar below our navigation.
+
+ProgressBar has two modes; `determinate` (default) and `indeterminate`. In determinate mode, a value between 0 and 100 is needed to display the progress. 
+
+We can also import and use a InputText component in our project, as highlighted below
+
+```javascript
+import { InputText } from 'primereact/inputtext';"
+```
+
+Like our ProgressBar we also import our InputText inside app.js file, then add it inside our div tag.
+ 
+ ```javascript
+<div classname="App"> 
+ <Navigation /> 
+ <br /> 
+ <ProgressBar value={value} /> 
+ <br />
+ <InputText />
+</div>. 
+```
+
+When you refresh your browser, you will see an empty input text under the progressbar.
+
+We can also import and use a Button component in our project. We first import in inside our app.js then add it to our div tag
+
+```javascript
+import { Button } from 'primereact/button';". 
+```
+
+Then we import it inside our div tag
+
+ ```javascript
+ <div classname="App"> 
+ <Navigation /> 
+ <br /> 
+ <ProgressBar value={value} /> 
+ <br />
+ <InputText/>
+ <br />
+ <Button label="Click Me" />
+</div>
+```
+
+If you go to your browser you will see a button with the words "Click Me" this button will not be styled. We can add more styles to the button by adding an i tag within it. You can read more about PrimeReact Button components from [here](https://primefaces.org/primereact/showcase/#/button)
+
+The Button element then becomes,
+
+```javascript
+ <Button label="Click Me" >
+ <i className="pi pi-spin pi-spinner" style={{'fontSize': '4px'}}></i>
+ </Button> 
+ ```
```

Comment:
What is the purpose of this section if your article is about "How to create a responsive React navigation bar using PrimeReact"? I think you should remove this and concentrate on the Navbar aspect of the article.